web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -30,7 +30,19 @@ function StorePlugin(props: StorePluginProps) {
 
   const argsStore = {
     preIcon: <img src={logo} alt="plugin-logo" />,
-    topAction: <FiDownload onClick={() => mutate({ params })} />,
+    topAction: (
+      <Button
+        customClass={isLoading ? 'animate-pulse blur-sm' : ''}
+        onClick={() => mutate({ params })}
+        disabled={isLoading}
+      >
+        <div
+          className={`rounded-full ${isLoading ? 'animate-pulse blur-sm' : ''}`}
+        >
+          <FiDownload />
+        </div>
+      </Button>
```

Comment:
I would prefer we extract to a file called `Loading` as we did for Search (and other) pages and here we just switch between them. isLoading ? <Loading/> : <...>

Reply:
done

Reply:
Cool! Thank you! 🎉